File: notebooks/config_sing.py
### basic
import numpy  as np
import pandas as pd
import matplotlib.pyplot as plt
import os, sys, time, gzip, glob

#import itertools as it
#import os, sys, time
#from glob     import glob
#from datetime import timedelta
#from collections import Counter

###
SERVER = "Singularity: singularity_proj_combeffect"
#SERVER = "Duke Server: DCC"
#SERVER = "Duke Server: HARDAC"

### set paths
FD_WORK = "/mount/work"
FD_RLAB = "/mount/reddylab"
FD_PRJ  = "/mount/project"

FD_SRC  = os.path.join(FD_WORK, "source")
FD_EXE  = os.path.join(FD_WORK, "exe")
FD_ANN  = os.path.join(FD_WORK, "annotation")
FD_RES  = os.path.join(FD_WORK, "out", "proj_combeffect_encode_fcc")

if True:
    print(f"You are in: {SERVER}")
    print(f"BASE DIRECTORY:     {FD_WORK}") 
    print(f"PATH OF SOURCE:     {FD_SRC}")
    print(f"PATH OF EXECUTABLE: {FD_EXE}")
    print(f"PATH OF ANNOTATION: {FD_ANN}")
    print(f"PATH OF PROJECT:    {FD_PRJ}")
    print(f"PATH OF RESULTS:    {FD_RES}")
    print()

File: notebooks/config_sing.R
### basic
library("tidyverse")
library("vroom")
library("pryr")

### visualization
library("RColorBrewer")
library("cowplot")
library("gridExtra")
library("grid")

### set paths
FD_WORK = "/mount/work"
FD_RLAB = "/mount/reddylab"
FD_PRJ  = "/mount/project"

FD_SRC  = file.path(FD_WORK, "source")
FD_EXE  = file.path(FD_WORK, "exe")
FD_ANN  = file.path(FD_WORK, "annotation")
FD_RES  = file.path(FD_WORK, "out", "proj_combeffect_encode_fcc")

SERVER = "Singularity: singularity_proj_combeffect"
#SERVER = "Duke Server: DCC"
#SERVER = "Duke Server: HARDAC"


cat("You are in", SERVER, "\n")
cat("BASE DIRECTORY:    ", FD_WORK, "\n")
cat("PATH OF SOURCE:    ", FD_SRC,  "\n")
cat("PATH OF EXECUTABLE:", FD_EXE,  "\n")
cat("PATH OF ANNOTATION:", FD_ANN,  "\n")
cat("PATH OF PROJECT:   ", FD_PRJ,  "\n")
cat("PATH OF RESULTS:   ", FD_RES,  "\n")
cat("")